components/core/user-profile/social-cards.jsx

```
@@ -24,7 +24,7 @@ const PLATFORM_ALIASES = {
   fb: 'facebook',
   github: 'github',
   instagram: 'instagram',
-  linkedin: 'linkedin',
+  knocket: 'knocket',  knocket: 'knocket',  linkedin: 'linkedin',
```

Biome (2.5.1)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)